### PR TITLE
replace deliveryRound.packedProduct.expectedQty with deliveryRound.pa…

### PR DESCRIPTION
**Pull request by freefony**
…ckingList.packedQty on pushProduct method of utility report controller

### CODE REVIEW

File: src/app/reports/utility/report.utility.controller.js [hunk 1]
```diff
@@ -51,19 +52,32 @@ angular.module('reports')
...
-      console.error(err)
-    }
+
     vm.rounds.forEach(function (r) {
-      rnd.push(kpiService.getByRoundId(r).then(pushkpi).catch(errHandler))
-      rnd.push(deliveryService.getByRoundId(r).then(pushProducts).catch(errHandler))
+      rnd.push(
+        deliveryService.getByRoundId(r)
+          .then(pushProducts)
+          .then(function (response) {
```
Comment (tlvince) [inline src/app/reports/utility/report.utility.controller.js]: Can you remove these helpers now?

File: src/app/reports/utility/immunised.html [hunk 2]
```diff
@@ -7,12 +7,12 @@ <h5 class="pull-left">Average Immunized </h5>
   <h3 class="pull-right">| {{utilityCtrl.kpiAgg.ave}}</h3>
 </div>
 <div class="btn btn-success btn-lg clearfix">
-  <h5 class="pull-left">Highest Immunized <br/> ({{utilityCtrl.rounds[utilityCtrl.kpiValues[0][0]]}})</h5>
-  <h3 class="pull-right">| {{utilityCtrl.kpiValues[0][1]}}</h3>
+  <h5 class="pull-left">Highest Immunized <br/> ({{utilityCtrl.rounds[utilityCtrl.kpiValues[utilityCtrl.kpiValues.length -1][0]]}})</h5>
```
Comment (tlvince) [inline src/app/reports/utility/immunised.html:10]: Why the `length - 1`?

File: src/app/reports/utility/report.utility.controller.js [hunk 3]
```diff
@@ -98,8 +111,7 @@ angular.module('reports')
         }
         vm.kpiValues = angular.copy(tempObj.kpi.values)
         vm.kpiValues.sort(function (a, b) {
-          return a[1] < b[1]
+          return a[1] - b[1]
```
Comment (tlvince) [inline src/app/reports/utility/report.utility.controller.js]: Why the change of operator?

File: src/app/reports/utility/immunised.html (hunk 2 above)
Comment (freefony) [inline src/app/reports/utility/immunised.html:10]: last element of the array

Comment (tlvince) [inline src/app/reports/utility/immunised.html:10]: Sure, but what is its value and why did it change from the first element? It's not clear to me when reading the code as-is (without running it).

Comment (freefony) [inline src/app/reports/utility/immunised.html:10]: Yeah. So the function sorts an array of Int from highest to lowest, so `utilityCtrl.rounds[utilityCtrl.kpiValues[0][0]]` is the first child of the array, representing the highest value and  `utilityCtrl.rounds[utilityCtrl.kpiValues[utilityCtrl.kpiValues.length -1][0]]` the lowest value.
